#
# MISC
#
integral_size: 200
reset_kalman_threshold: 3.0

generalK: "/home/ubuntu/git/jetyak_uav_utils/cfg/generalK.txt"
landK: "/home/ubuntu/git/jetyak_uav_utils/cfg/generalK.txt"

#
# Takeoff
#
takeoff_height: 0
takeoff_threshold: .25

#
# Follow Parameters
#
follow_x: -2
follow_y: 0
follow_z: 2
follow_w: 0

#
# Land Parameters
#
land_x: -1.6
land_y: 0
land_z: .2 # this should be .1-.2m over the tag to allow the uav to maneuver
land_w: 0

land_velMag: .1
land_xThresh: .12
land_yThresh: .12
land_zThresh: .12
land_angleThresh: .15

#
# Return Parameters
#
return_gotoHeight: 10
return_heightThresh: 1
return_finalHeight: 4
return_downRadius: 1
return_settleRadius: .5
return_tagTime: 1
return_tagLossThresh: 1
return_maxVel: 3.0
